types/matchmediaquery/index.d.ts

```diff
@@ -0,0 +1,37 @@
+// Type definitions for matchmediaquery 0.3
+// Project: https://github.com/ncochard/matchmediaquery#readme
+// Definitions by: Nick Whitlock <https://github.com/eezstreet>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+type MediaValues = Record<
+    | 'orientation'
+    | 'scan'
+    | 'width'
+    | 'height'
+    | 'device-width'
+    | 'device-height'
+    | 'resolution'
+    | 'aspect-ratio'
+    | 'device-aspect-ratio'
+    | 'grid'
+    | 'color'
+    | 'color-index'
+	| 'monochrome'
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/matchmediaquery/index.d.ts;
  Line: 19;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.